desktop/app/kbfs-helper.js

```diff
@@ -1,10 +1,32 @@
 // @flow
 
 import {ipcMain, shell} from 'electron'
+import fs from 'fs'
+import {pathToURL} from './paths'
 
 export default function () {
   ipcMain.on('openInKBFS', (e, path) => {
-    // We do the `shell.openItem` here to not block the ui thread
-    shell.openItem(path)
+    fs.stat(path, (err, stats) => {
```

Comment:
Yeah it looks like KBFS will wait for identify which may have to wait on HTTP requests... I experienced it here, when coinbase took a while to resolve:
https://keybase.atlassian.net/browse/CORE-3488

It seems like the OS beach-balling while keybase is verifying a coinbase proof is not desired behavior?!